### PR TITLE
Add Exchange-API quick start document

### PR DESCRIPTION
**Pull request by omordyk**
Add Exchange-API quick start document.
**Description:**
This document describes the environment and the set of steps aiming to create a local development and debugging environment.

### CODE REVIEW

File: docs/QuickStart.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+* [Java 17](https://www.java.com/en/)
+* [SBT](https://www.scala-sbt.org/)
+* [PostgreSQL](https://www.postgresql.org/)
+
+## Installation
+
+### Create a new user (optional)
+For development purposes recommended not to use root user. Command below will create
+a new Ubuntu user.
+```
```
Comment (johnwalicki) [inline docs/QuickStart.md:33]: Can you add ```bash to each of the markdown code blocks so that highlighting is correct for bash commands?

**Review by johnwalicki — CHANGES_REQUESTED** [review]
Can you add ```bash to each of the markdown code blocks so that highlighting is correct for bash commands?

File: docs/QuickStart.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+postgres=# CREATE DATABASE horizon OWNER new_user;
+\q
+```
+
+### Test connection to DB
+```
+psql "host=localhost dbname=horizon user=kevin password='password'"
+```
+
+
```
Comment (johnwalicki) [inline docs/QuickStart.md:82]: Lint warning - remove this extra line of whitespace.

File: docs/QuickStart.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+### Clone exchange-api and run it:
+```
+https://github.com/open-horizon/exchange-api.git
+
+~/exchange-api$ sbt
+
+sbt:amd64_exchange-api> reStart
+```
```
Comment (johnwalicki) [inline docs/QuickStart.md:98]: Lint warning - add a trailing line of whitespace

**Review by johnwalicki — CHANGES_REQUESTED** [review]

**Comment by johnwalicki** [comment]
Great installation page.  If the goal will be to move this under the Open Horizon documentation website, it should complement the existing page (which has some of these instructions already)
https://open-horizon.github.io/docs/exchange-api/docs/README/

**Comment by omordyk** [comment]
@johnwalicki Initial idea was to create a small guide for newcomers. I summarize my steps for preparing local development environment. If it is necessary I can move these steps in main Readme.

**Review by johnwalicki — APPROVED** [review]
Thanks for the lint fixes. Approved.

**Comment by johnwalicki** [comment]
I like the QuickStart.md doc in your PR.  It contains additional details which are not in the main [README.md](https://github.com/open-horizon/exchange-api/blob/master/docs/README.md)  Some overlap however.

We should ask @joewxboy to also review.  We could merge these details into the README.md, accept this PR with its duplication, or split the Exchange API doc into an Installation guide (adding your details) and a Run/Tips/Docker guide.

**Comment by joewxboy** [comment]
@johnwalicki and @naphelps if no objections, how about merging then polishing as a separate task?

**Comment by naphelps** [comment]
No Objections.

**Pull request merged**